### PR TITLE
splits out tgui from the debug master category

### PR DESCRIPTION
**Pull request by ZephyrTFA**
## About The Pull Request

Splits out tgui from the debug master category at the request of @Fikou 
Removes pointless overrides of internal_format since the default is now FALSE and also adds an additional comment about ensuring the shit works if you change the default.

## Why It's Good For The Game

I believe the original reason was that fikou didnt want to sort through the entire debug log for tgui stuff.

### CODE REVIEW

File: code/__DEFINES/logging.dm [hunk 1]
```diff
@@ -95,6 +95,7 @@
 #define LOG_CATEGORY_MANIFEST "manifest"
 #define LOG_CATEGORY_MECHA "mecha"
 #define LOG_CATEGORY_PAPER "paper"
+#define LOG_CATEGORY_QDEL "qdel"
```
Comment (MrMelbert) [inline code/__DEFINES/logging.dm:98]: why're you de-alphabetizing this?

File: code/__HELPERS/logging/ui.dm [hunk 2]
```diff
@@ -1,12 +1,17 @@
...
  */
-/proc/log_tgui(user, message, context,
-		datum/tgui_window/window,
-		datum/src_object)
+/proc/log_tgui(
+	user,
+	message,
+	context,
+	datum/tgui_window/window,
+	datum/src_object
```
Comment (JohnFulpWillard) [inline code/__HELPERS/logging/ui.dm:12]:
```suggestion
	datum/src_object,
```

**Review by JohnFulpWillard — COMMENTED** [review]
respond to melbert's comment please

File: code/__DEFINES/logging.dm (hunk 1 above)
Comment (ZephyrTFA) [reply code/__DEFINES/logging.dm:98]: I'm not? This is making it alphabetical

**Comment by ZephyrTFA** [comment]
> respond to melbert's comment please

Didnt respond to it because it's wrong.

**Pull request merged**